server/user-update.js

```diff
@@ -63,6 +63,7 @@ module.exports = function handle({ message = {} }, { ship, hull }) {
     }
 
     if (errors && errors.length > 0) {
+      hull.post(`/connector/${ship.id}/notifications`, { status: "error", message: "Script error" });
```

Comment:
Probably want to post as User -> this way we can store the user Id in the notification and later easily surface it in the user profile by retrieving those messages